### ISSUE TITLE
SVGs handling

### ISSUE THREAD

**Issue by bdo** [EurosportDigital/web-toolkit#27]
We currently cannot use SVGs by themselves right now as we are not using a module bundler.

Probably depends on #8 unless we go with [parcel](https://parceljs.org/) or [rollup](https://rollupjs.org/).

**Comment by bdo** [comment]
duplicate of #8
